vale/styles/jargonLint/Licenses.yml

```diff
@@ -0,0 +1,41 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+message: Use '%s' instead of '%s'
+action:
+  name: replace
+swap:
+  Apache[ -]?2(?:\.0)?:Apache 2.0
+  [0O]BSD: 0BSD
+  BSD: BSD
```

Comment:
This one may collide with BSD the operating system. Not a real problem, but I'm worried about BSD versions. I don't have any real examples for now

Reply:
But 

- BSD-licensed
- BSD licensed
- BSD license
- BSD licenses
- BSD licensed
- license BSD
- licenses BSD
- licensed BSD
- dual-licensed BSD

These are license based, so can be added to the file by using look behind or forward regexp.

Funny thing about license/license US/UK

So we have to duplicate with licence

BSD the OS, can also be found with slash

Linux/BSD

So maybe we will have to move terms with multiple meaning to a separate file

Reply:
to be addressed by #42